deploy/helm/kubecf/templates/ingress.yaml

```diff
@@ -167,4 +167,40 @@ spec:
   clusterIP: {{ $service.clusterIP | quote }}
   {{- end }}
 {{- end }}
+---
```

Comment:
Do we have nothing when ingress is on?

There's some docs for ssh in [`doc/dev/general.md`] / [`doc/dev/minikube.md`] that could be updated.

[`doc/dev/general.md`]: https://github.com/SUSE/kubecf/blob/b5cf7622a5c78ec5395649db1d6a8894788b7dcd/doc/dev/general.md#installation-of-the-nginx-ingress-controller
[`doc/dev/minikube.md`]: https://github.com/SUSE/kubecf/blob/b5cf7622a5c78ec5395649db1d6a8894788b7dcd/doc/dev/minikube.md#installation-of-the-nginx-ingress-controller

Reply:
True, I need to update the ingress docs. Thanks.